Mathlib/GroupTheory/Index.lean

```diff
@@ -392,6 +392,23 @@ noncomputable def fintypeOfIndexNeZero (hH : H.index ≠ 0) : Fintype (G ⧸ H)
 lemma index_eq_zero_iff_infinite : H.index = 0 ↔ Infinite (G ⧸ H) := by
   simp [index_eq_card, Nat.card_eq_zero]
 
+@[to_additive (attr := gcongr)]
+lemma index_antitone (h : H ≤ K) [h₁ : Finite (G ⧸ H)] : K.index ≤ H.index := by
```

Comment:
You can use the `Subgroup.FiniteIndex` class here.

Reply:
Good idea, thanks.